### ISSUE TITLE
Windows defender detects v1.8.1.10 as a Trojan

### ISSUE THREAD

**Issue by Ajedi32** [nicehash/NiceHashMiner#1090]
The latest release of NHML gets detected as a Trojan and quarantined by Windows Defender.

![image](https://user-images.githubusercontent.com/1876931/37561981-dbaad650-2a2a-11e8-98cd-d540f3a43ec3.png)

Details: https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan:Win32/Skeeyah.G

My first instinct is to assume this is a false positive, as crypto mining malware is rather prevalent these days. However, I don't want to rule out the possibility of the NHML official builds being corrupted somehow. (Casual miners are a valuable target, as they are likely to have cryptocurrency wallets on their machines.)

These claims should be investigated and if it is found to be a false positive, we should determine how best to get Windows Defender to stop complaining.

**Comment by tdkavngr** [comment]
I also have this problem, whens the fix guys??????

**Comment by Gotitalready** [comment]
Windows defender detected Trojan:Win32/Skeeyah.G in 1.8.1.10 also, but not in 1.8.1.8; it is classified as severe.  "nerdatwork" in another thread suggested changing virus scanners.  Rather than change your virus scanner, go back to using 1.8.1.8.  I plan on doing that.  nerdatwork's suggestion is like saying if a CT scan detected a canerous growth inside of you then change test to something like a blood test so that you won't get a warning that you have cancer.
UPDATE:  After removing all the files and folders that Windows defender detected as trojans, including downloaded zip files and hidden files, doing another complete scan came up with NO more trojans.  I am now hashing with the 1.8.1.8 version and no problems with it.
Why trust these programmers who may steal your coins if you have them stored on your computer hard drive?  Worse yet, they could install a program to use your graphics card to mine for them!!

**Comment by allstarfxmaster** [comment]
https://github.com/nicehash/NiceHashMinerLegacy/wiki/Troubleshooting#what-if-i-do-not-feel-safe-with-making-av-exceptions

**Comment by Ajedi32** [comment]
Yeah, that's what I figured. (Though it's unfortunate that AV false positives are WON'TFIX for NHML.)

Still, this didn't happen with the previous version of the software, so it'd be nice to know what changed that's causing this particular warning.

**Comment by Boricuatec** [comment]
False positives are hard to track down what may be a false positive today maybe okay tomorrow because they just their filters and vice-versa I agree with the engineers on this this is more on the antivirus side maybe some communication should be opened up with them to allow for it


Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Andrew Meyer <notifications@github.com> Date: 3/18/18  10:03 AM  (GMT-08:00) To: nicehash/NiceHashMinerLegacy <NiceHashMinerLegacy@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [nicehash/NiceHashMinerLegacy] Windows defender detects v1.8.1.10 as a Trojan (#1090) 
Yeah, that's what I figured. (Though it's unfortunate that AV false positives are WON'TFIX for NHML.)
Still, this didn't happen with the previous version of the software, so it'd be nice to know what changed that's causing this particular warning.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/nicehash/NiceHashMinerLegacy","title":"nicehash/NiceHashMinerLegacy","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/nicehash/NiceHashMinerLegacy"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Ajedi32 in #1090: Yeah, that's what I figured. (Though it's unfortunate that AV false positives are WON'TFIX for NHML.)\r\n\r\nStill, this didn't happen with the previous version of the software, so it'd be nice to know what changed that's causing this particular warning."}],"action":{"name":"View Issue","url":"https://github.com/nicehash/NiceHashMinerLegacy/issues/1090#issuecomment-374016392"}}}
